torchmetrics/functional/text/bert.py

```diff
@@ -530,14 +533,22 @@ def bert_score(
             A path to the user's own local csv/tsv file with the baseline scale.
         baseline_url:
             A url path to the user's own  csv/tsv file with the baseline scale.
+        predictions:
+            Either an iterable of predicted sentences or a `Dict[str, torch.Tensor]` containing `input_ids` and
+            `attention_mask` `torch.Tensor`.
+            This argument is deprecated in v0.7 and will be removed in v0.8. Use `preds` instead.
+        references:
+            Either an iterable of target sentences or a `Dict[str, torch.Tensor]` containing `input_ids` and
+            `attention_mask` `torch.Tensor`.
+            This argument is deprecated in v0.7 and will be removed in v0.8. Use `target` instead.
 
     Returns:
         Python dictionary containing the keys `precision`, `recall` and `f1` with corresponding values.
 
     Raises:
         ValueError:
-            If `len(predictions) != len(references)`.
-        ModuleNotFoundError:
+            If `len(preds) != len(target)`.
+        ValueError:
```

Comment:
```suggestion
        ModuleNotFoundError:
```
or any reason for reverting it to `ValueError`?

Reply:
That was changed by mistake. I'll revert that